exla/Makefile

```diff
@@ -28,11 +28,15 @@ TENSORFLOW_DIR = $(EXLA_CACHE)/$(TENSORFLOW_NS)/erts-$(ERTS_VERSION)
 TENSORFLOW_EXLA_NS = tensorflow/compiler/xla/exla
 TENSORFLOW_EXLA_DIR = $(TENSORFLOW_DIR)/$(TENSORFLOW_EXLA_NS)
 
-all: symlinks
+XLA_SO = $(TENSORFLOW_DIR)/bazel-bin/$(TENSORFLOW_EXLA_NS)/libexla.so
+
+all: symlinks $(XLA_SO)
+	mkdir -p priv
+	cp -f $(XLA_SO) $(EXLA_SO)
+
+$(XLA_SO): $(TENSORFLOW_DIR)
```

Comment:
Thanks @hauleth!

I believe this is a welcome change but unfortunately it will break our development experience and code updates because if we change exla.cc or any other file, it won't be rebuilt. And unfortunately the only way to rebuild it is by asking bazel.

Reply:
Also, I believe the reason ElixirLS is building again is because Bazel is detecting it is being built in a different environment, and builds it from scratch. We should try to find a way to address this without breaking the development flow. Does ELIXIR_LS set an environment variable? Maybe we can execute the code you added here if said env var is set.